src/lib/realmdb/RealmDB.js

```diff
@@ -89,17 +76,20 @@ class RealmDB implements DB, ProfileDB {
   async _initRealmDB() {
     const REALM_APP_ID = Config.realmAppID
     const jwt = await AsyncStorage.getItem(JWT)
-    log.debug('initRealmDB', { jwt, REALM_APP_ID })
     const credentials = Realm.Credentials.jwt(jwt)
+
+    log.debug('initRealmDB', { jwt, REALM_APP_ID })
+
     try {
       // Authenticate the user
       const app = new Realm.App({ id: REALM_APP_ID })
-      this.user = await app.logIn(credentials)
+      const user = await app.logIn(credentials)
       const mongodb = app.currentUser.mongoClient('mongodb-atlas')
-      this.database = mongodb.db(this._databaseName)
 
       // `App.currentUser` updates to match the logged in user
-      log.debug('realm logged in', { user: this.user })
+      log.debug('realm logged in', { user })
+      assign(this, { user, database: mongodb.db(this._databaseName) })
```

Comment:
i dont like this syntax. it is very hard to
1. find out where a class variable is assigned (you cant search for 'this.X')
2. also the destructring syntax is confusing { Feed } = this, again hard to search for all palces where this.Feed is used